packages/embark-env/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "embark-env",
+  "version": "4.1.0-beta.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Embark Env utilities",
+  "homepage": "https://github.com/embark-framework/embark/tree/master/packages/embark-env#readme",
+  "bugs": "https://github.com/embark-framework/embark/issues",
+  "keywords": [
+    "blockchain",
+    "dapps",
+    "ethereum",
+    "ipfs",
+    "serverless",
+    "solc",
+    "solidity"
+  ],
+  "files": [
+    "dist"
+  ],
+  "license": "MIT",
+  "repository": {
+    "directory": "packages/embark-env",
+    "type": "git",
+    "url": "https://github.com/embark-framework/embark.git"
+  },
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "cross-env BABEL_ENV=node babel src --extensions \".js,.ts\" --out-dir dist --root-mode upward --source-maps",
+    "ci": "npm run qa",
+    "clean": "npm run reset",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint src/",
```

Comment:
As seen in CI, eslint is unhappy because there aren't any `.js` files in `src/` (only `.ts`). So the `"lint:js":` script can be commented out like `"// lint:js":` or simply removed. If removed, then L45-47 can also be removed.

Reply:
Whoops, I just realized I commented in a line in `embark-env` not `embark-fs`, but it's the same situation for both.